app/controllers/friendships_controller.rb

```diff
@@ -5,6 +5,7 @@ class FriendshipsController < ApplicationController
   before_action :set_user
```

Comment:
I think user sign up is broken as well.  I am looking into all of these changes.  Just very slowly